Update readme (#51)
RockefellerArchiveCenter/request_broker: README.md

# Request Broker

> An application that accepts requests with lists of ArchivesSpace URIs from users of our discovery system and for each item grabs data from ArchivesSpace. Formats that data for delivery to our item retrieval system (Aeon) for retrieval or digital duplication, or formats the data for email delivery or CSV download. Routes formatted data to retrieval management system or to an email for researcher use.

Reviewer: ```suggestion
An application that accepts requests with lists of ArchivesSpace URIs from users of our discovery system. For each item, the application fetches data from ArchivesSpace, formats that data for delivery to our retrieval management system (Aeon) to enable reading room or duplication requests, or formats the data for email delivery or CSV download. It routes formatted data to the retrieval system or to an email for researcher use. This is a passthrough service; users cannot create requests directly in the application.
```

> ## Services

Reviewer 2: There's also the CSV Download service that's missing.

> * Iterates over a list of request URIs and gets corresponding data from ArchivesSpace and parses over the data and marks it as submittable or unsubmittable.

Reviewer 2: ```suggestion
* Request Pre-Processing: Iterates over a list of request URIs, fetches corresponding data from ArchivesSpace, parses the data and marks it as submittable or unsubmittable.
```

> * Converts AS data into strings and it to the message body of an email and then sends an email to an address or list of addresses.

Reviewer 2: ```suggestion
* Mailer: correctly formats the body of an email message and sends an email to an address or list of addresses.
```

> * Creates retrieval and duplication transactions in Aeon by sending data to the Aeon API.

Reviewer 2: ```suggestion
* Aeon Request Submission: creates retrieval and duplication transactions in Aeon by sending data to the Aeon API.
```
